packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+# 49.6.0 - August 20, 2020
+
+## Fixed
+
+#### salesforcedx-vscode-core
+
+- Remove files in an existing project when running the `SFDX: Create and Set Up Project for ISV Debugging` command ([PR #2407](https://github.com/forcedotcom/salesforcedx-vscode/pull/2407))
```

Comment:
**Nit:** Prompts the user to remove stale files in an existing project when running...